### ISSUE TITLE
Some of color editor inputs do not have neither labels nor ARIA labels

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#37]
**User Experience:**
If label is not defined for edit field than screen reader user will not be able to find out what kind of information is needed in the edit fields and will face difficulties while navigating on the page.

**Repro-Steps:**
1. Open Azure portal with valid credentials and create a API Management.
2. Select Developer Portal (Preview) button and select it.
3. Navigate to the https://accessibility.developer.azure-api.net/.
4. Navigate to the Left nav and select Styles  button.
5. Navigate to style page and select Edit Color  button.
6. Open Accessibility insights for web tool and select past pass.
7. Verify the issue

**Actual Result:**
Label is not defined for the input fields, present  inside 'Add color' popup.

**Expected result:**
Label should be defined for the input field under 'Add color'.

Issue: Form elements must have labels (label: https://dequeuniversity.com/rules/axe/3.2/label?application=msftAI)

Target application title: API Management
Target application url: https://accessibility.developer.azure-api.net/

Element path: `.form-control`

Snippet:
`<input type="text" class="form-control" spellcheck="false" data-bind="textInput: colorName">`

**How to fix:**
Fix any of the following:
-   aria-label attribute does not exist or is empty
-   aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
-   Form element does not have an implicit (wrapped) <label>
-   Form element does not have an explicit <label>
-   Element has no title attribute or the title attribute is empty

This accessibility issue was found using Accessibility Insights for Web 2.6.0 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
